### DISCUSSION TITLE
SSL KEY LOG

### DISCUSSION THREAD

**Discussion by arabian9009** [mitmproxy/mitmproxy#4987] [Q&A]
How to add SSL KEY LOGFILE into MITM proxy Windows. Will it be possible to describe?

Thanks.

**Comment by mhils** [comment]
https://docs.mitmproxy.org/dev/howto-wireshark-tls/

**Reply by arabian9009** [reply]
How to add ssl key log on Windows. 
When I Use the MITM proxy on Linux Ubuntu. It exports SSL key log file but when I  add it to Wireshark it is not decrypting traffic
Version of Mitm Proxy7.0 . Open SSL 1.1.1
